### ISSUE TITLE
Fix requred to github action example

### ISSUE THREAD

**Issue by fr0stnk** [hoangvvo/gitflow-workflow-action#36]
When used example "Integration: Post to Slack" received "Missing required root key on"
![image](https://github.com/hoangvvo/gitflow-workflow-action/assets/10252136/f5fa452e-89a9-44d7-94d0-8a581b588fc8)

**Comment by jonathan8devs** [comment]
Of course, the 'on' is missing in this snippet. The documentation states that it is an example of the job and not a stand-alone action
